src/mlpack/methods/ann/rbm/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+# Define the files we need to compile.
+# Any file not in this list will not be compiled into mlpack.
+set(SOURCES
+  rbm.hpp
```

Comment:
hm, not sure creating an extra directory is what we should do here, I think we could either move the directory in the method folder or move the rbm class into the ann folder. Let me know what you think.

Providing an executable for the rbm class is a lot easier since the model structure is somewhat static.

Reply:
This was done just to be in line with our gan implementation, but I think, since `GAN`s and `RBM`s are sub-variants of `ANN`s it makes sense to keep a folder here. I'm open to suggestions though.

Reply:
I get your point, so I guess if we are going to write an executable it's in the rbm folder as well?

Reply:
Yes.